hourglass.py

```diff
@@ -36,22 +36,25 @@ def build_model(self, mobile=False, show=False):
         if show:
             self.model.summary()
     
-    def load_and_filter_annotations(self, path_to_train_anns, path_to_val_anns):
+    def load_and_filter_annotations(self, path_to_train_anns,path_to_val_anns, subset):
         df = coco_df.get_df(path_to_train_anns,path_to_val_anns)
         # apply filters here
         print(f"Unfiltered df contains {len(df)} anns")
         df = df.loc[df['is_crowd'] == 0] # drop crowd anns
         df = df.loc[df['num_keypoints'] != 0] # drop anns containing no kps
         df = df.loc[df['bbox_area'] > BBOX_MIN_SIZE] # drop small bboxes
-        print(f"Filtered df contains {len(df)} anns")
         train_df = df.loc[df['source'] == 0]
         val_df = df.loc[df['source'] == 1]
+        if subset != 1.0:
+            train_df = train_df.sample(frac=subset)
+            val_df = val_df.sample(frac=subset)
```

Comment:
personally I think we should keep validation unchanged, because the validation run doesn't actually take that long (less than 2 minutes) since it's not updating any weights. That way, we can compare validation scores across different parameters.

Reply:
Hmm, @robertklee do you think it's worth it to have val subset AND train subset arguments? This way if someone so desires, they can perform really fast training with small val as well. If not then I'll just remove the val subset line.

Reply:
I think validation takes so little time that it's probably fine as is. Inference is a lot faster typically.